department-of-veterans-affairs/va.gov-cms
# VACMS-5596: Enable media standalone URL so that viewing/editing/locking work as expected.

## Description

Closes #5596.

Media is currently "edit-only".  You can't just look at an image -- you have to edit it.

Viewing media without editing it can be enabled by checking this checkbox:

![Screen Shot 2022-09-08 at 7 57 14 AM](https://user-images.githubusercontent.com/1318579/189116156-13cbaa65-7fec-4f3c-8c2b-e49a1c25f33e.png)

If it's checked, then you can view media.  And, consequently, when you unlock the content that you're editing, you will be redirected to a normal view of the media item.

I think this is the best approach to solving the aggravating problem with content lock on media items.  It doesn't involve any custom code or otherwise add technical debt, and makes handling media more consistent with how we handle nodes.

## Testing done

Verified that I could view media without editing it, and that unlocking a media item I was editing redirected me to the view page for the media item, rather than back to the edit page.

## Screenshots


## QA steps

As a user who can edit media items:

1. Visit this page: https://cms-9x4r85kz3xkp6gem8b8wjarqvtiow7lu.ci.cms.va.gov/media/image/4165
  - [ ] Confirm that this is an "Edit" form for the media in question
  - [ ] Confirm that the media is locked: 
![Screen Shot 2022-09-08 at 8 07 23 AM](https://user-images.githubusercontent.com/1318579/189117862-53cee911-7cd5-43dc-87f0-049194a80b97.png)
2. Click the "Unlock" button.
3. Click the "Confirm break lock" button.
  - [ ] Confirm that you are at this URL: https://cms-9x4r85kz3xkp6gem8b8wjarqvtiow7lu.ci.cms.va.gov/media/4165
  - [ ] Confirm that you see the media item, but are not _editing_ it.
4. Click the "Edit" button.
  - [ ] Confirm that this is an "Edit" form for the media in question
  - [ ] Confirm that the media is locked: 
![Screen Shot 2022-09-08 at 8 07 23 AM](https://user-images.githubusercontent.com/1318579/189117862-53cee911-7cd5-43dc-87f0-049194a80b97.png)
